### ISSUE TITLE
Do not capitalize titles for PowerShell helpers or for select Validator rules

### ISSUE THREAD

**Issue by TheCakeIsNaOH** [chocolatey/docs#627]
## What New Or Updated Would You Like To See?

That some page titles are should be changed to be no longer completely uppercase. (or should be otherwise changed that they can be more easily parsed)

## Why Is It Needed?

There are some validator rules that refer to case-sensitive `.nuspec` elements in the title. For example, in the `.nuspec` it needs to be  `ProjectSourceUrl` not `PROJECTSOURCEURL`
![image](https://user-images.githubusercontent.com/38865330/208504907-8e33a61b-78e2-4adb-b9cc-ad841c4c32dc.png)
https://docs.chocolatey.org/en-us/community-repository/moderation/package-validator/rules/cpmr0041#mainContent

----

And there are some PowerShell helpers that are hard to parse into individual words without the normal casing, see `Install-ChocolateyPowershellCommand` vs `INSTALL-CHOCOLATEYPOWERSHELLCOMMAND`
![image](https://user-images.githubusercontent.com/38865330/208505215-b2732ce2-4a03-403a-9a54-06b2b1693a92.png)
https://docs.chocolatey.org/en-us/create/functions/install-chocolateypowershellcommand#syntax

## Additional Context?

As brought up here: https://github.com/chocolatey/choco/issues/379#issuecomment-1344738411

**Comment by st3phhays** [comment]
This will actually be fixed with the release of choco-theme 0.2.0 which is set to be released in the beginning of the new year. No titles will be capitalized anymore by default.

**Comment by st3phhays** [comment]
This has been fixed with the deployment of choco-theme 0.2.0! See https://github.com/chocolatey/docs/pull/613
